### ISSUE TITLE
Custom Field List of Images use 3 /// in the image img src path

### ISSUE THREAD

**Issue by AlexRed** [joomla/joomla-cms#16121]
### Steps to reproduce the issue
Create a new Custom Field List of Images for the Content. Create a new article and select an image for the field.
See the article in frontend and see the image path: 
src="/Joomla_3.7.1/images**///**image-name.gif"

### Expected result
only a / in the the image img src path

### Actual result
3 /// in the image img src path

### System information (as much as possible)
Joomla 3.7.1

**Comment by infograf768** [comment]
Confirmed

**Comment by infograf768** [comment]
this can be solved by correcting
`/plugins/fields/imagelist/tmpl/imagelist.php`

Change
```
	$buffer .= '<img src="images/' . $fieldParams->get('directory', '/')
				. '/' . htmlentities($path) . '"' . $class . '/>';
```

to
```
	if ($fieldParams->get('directory', '/') !== '/')
	{
		$buffer .= '<img src="images/' . $fieldParams->get('directory', '/')
					. '/' . htmlentities($path) . '"' . $class . '/>';
	}
	else
	{
		$buffer .= '<img src="images/' . htmlentities($path) . '"' . $class . '/>';
	}
```

**Comment by infograf768** [comment]
@Bakual 
Would that fit?

**Comment by infograf768** [comment]
@laoneo ?

**Comment by AlexRed** [comment]
Ok for me the code by JM
Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16121">issues.joomla.org/tracker/joomla-cms/16121</a>.</sub>

**Comment by PhilETaylor** [comment]
Please test https://github.com/joomla/joomla-cms/pull/16135

**Comment by franz-wohlkoenig** [comment]
closed as having PR #16135<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16121">issues.joomla.org/tracker/joomla-cms/16121</a>.</sub>
